meshapp/src/components/SideMenu.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import {
+  Drawer,
+  Grid,
+  IconButton,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  Toolbar,
+} from "@mui/material";
+import {
+  TableRows as MenuIcon,
+  Notifications as NotificationsIcon,
+  Home as HomeIcon,
+  Receipt as ProfileIcon,
+  Inbox as MessagingIcon,
+  People as SwipingIcon,
+  Settings as SettingsIcon,
+  Logout as LogoutIcon,
+} from "@mui/icons-material";
+
+type ListItemTypes = { text: String; Icon: React.ElementType };
+
+// TODO: Add routes to each menu item when routing is implemented
```

Comment:
If youre going to add a ToDo, add a corresponding issue

Reply:
I have it linked in my pr description